Mapping/Proxy/ProxyFactory.php

```diff
@@ -101,7 +161,10 @@ private static function getHeader(array $options)
 
 use $base as Base;
 
-class $class extends Base
+/**
+ * DO NOT EDIT THIS FILE - IT WAS CREATED BY ELASTICSEARCH PROXY FACTORY.
+ */
+class $class extends Base implements \ONGR\ElasticsearchBundle\Mapping\Proxy\ProxyInterface
```

Comment:
If we are using `Base`, maybe we should check that project document doesn't use `Base` as the class name before proxy class generation.